streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/core/common/AbstractId.java

```diff
@@ -1,24 +1,25 @@
 package io.ray.streaming.runtime.core.common;
 
 import com.google.common.base.MoreObjects;
-import io.ray.streaming.runtime.core.resource.ContainerID;
+import io.ray.streaming.runtime.core.resource.ContainerId;
 import java.io.Serializable;
 import java.util.UUID;
 
 /**
- * Streaming system unique identity base class. For example, ${@link ContainerID }
+ * Streaming system unique identity base class.
+ * For example, ${@link ContainerId}
  */
-public class AbstractID implements Serializable {
+public class AbstractId implements Serializable {
```

Comment:
A think change like this is not related with this PR. Please revert it.

Reply:
I submit another PR to do this change, please have a look: https://github.com/ray-project/ray/pull/10523

Reply:
merged.